daemon/api.go

```diff
@@ -173,11 +173,11 @@ func sysInfo(c *Command, r *http.Request) Response {
 type metarepo interface {
 	Details(string, string) ([]snappy.Part, error)
 	All() ([]snappy.Part, error)
-	Updates() ([]snappy.Part, error)
+	Find(string) ([]snappy.Part, error) // XXX: change this to Search iff aliases (and thus SharedNames) are no more
```

Comment:
Can we have a slightly less enigmatic comment here? :-)

The note was useful for me to go digging further, but it'd be nice to be more clear here. Is that about the fact we have a Search method already? If so, can we have slightly more clear names for the time being, which highlight the distinction between Find and Search?